lib/compat/wordpress-6.6/class-gutenberg-rest-templates-controller-6-6.php

```diff
@@ -117,4 +117,159 @@ public function get_template_fallback( $request ) {
 
 		return rest_ensure_response( $response );
 	}
+
```

Comment:
If we are not landing this in 6.6, it should be in `lib/experimental` or `compat/wordpress-6.7` folder. And all @since tags should be updated too.

Reply:
Good point, @cbravobernal! For now, I rebased this PR and updated all files and `@since` tags to target 6.7.

If we prefer, I'm also happy to update it to be in `lib/experimental` instead. It's not 100% clear to me how the decision is made between one or the other, so happy to learn more about this. :slightly_smiling_face:

Reply:
Usually, if you want a feature to be considered for the next release, you move it to the `compat/wordpress-x.x` folder.

If you want a feature that is in the last WP version, but still, you want people with previous WP versions and Gutenberg to be able to test it, you should have it in `compat/wordpress-6.5`. We do that to prevent errors due to duplicated functions or calling non existent. Imagine calling a function that is only in 6.6, and an user has 6.5 with latest Gutenberg version, you don't want a crash there.

If you think your feature is quite experimental, and you still don't know in which version should land, then `lib/experimental` is a good place.

Reply:
Thanks for the explanation, @cbravobernal, that was very helpful! :raised_hands: